core/server/worker/src/main/java/alluxio/worker/netty/DataServerWriteHandler.java

```diff
@@ -150,6 +150,11 @@
      * Cancels the request.
      */
     abstract void cancel() throws IOException;
+
+    /**
+     * Cleans up the state.
```

Comment:
Could you add detail about when each of `close`, `cancel`, and `cleanup` should be called?